CONTRIBUTING.md

```diff
@@ -0,0 +1,132 @@
+Any change or upgrade that increases the project capabilities, performance or scalability.
+In general, enhancements include:
+    - Additional functionality
+    - Error/bug repair and handling
+    - Better code coverage
+    - Improved performance
+
+- ![Help Wanted](https://img.shields.io/badge/-help%20wanted-%23128a0c) *Indicates that a maintainer or developer wants help on an issue or pull request.*
+If you have a particular skill that can help in solving the problem then consider reading through these issues as they are a great place to offer your expertise.
+Not everyone can do everything. This is the time, the community needs your help so you better dive in!
+
+- ![Invalid](https://img.shields.io/badge/-invalid-%239a9a9a) *It indicates that an issue or pull request is no longer relevant.*
+
+- ![Question](https://img.shields.io/badge/-question-%23cc317c) *It indicates that an issue or pull request needs more information.*
+These issues are some questions regarding the project that might need clarifications and a bit of discussion. 
+
+## 3. Create a pull request :zap: 
+
+Pull requests with patches, improvements and new features are a great help. Please keep them clean from unwanted commits (One Commit One PR Rule)
+
+Follow the steps to get your work included in the project.
+
+#### 1. Fork the project, clone your fork, and add the BIDS Apps remote:
+
+```sh
+# Clone your fork of the repo into the current directory
+git clone https://github.com/<USERNAME>/bids-apps.github.io.git
+
+# Navigate to the cloned directory
+cd bids-apps.github.io
+
+# Assign the original repo to a remote called "upstream"
+git remote add upstream https://github.com/BIDS-Apps/bids-apps.github.io.git
+```
+#### 2. Get the latest changes from upstream:
+
+```sh
+git checkout master
+git pull upstream master
+```
+
+#### 3. Create a new branch from the main master branch to contain your changes:
+
+```sh 
+git checkout -b <topic-branch-name> 
+```
+
+#### 4. Locally rebase the upstream master branch into your topic branch:
+
+```sh
+git pull --rebase upstream master 
+```
+Push your topic branch up to your fork:
+```sh
+git push origin <topic-branch-name> 
+```
+
+#### 5. Open a Pull Request with a clear title and description against the master branch.
+
+[Open a Pull Request.](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request)
+
+A maintainer/developer will review and might suggest some changes before merging them into the codebase.
+
+Success!! :tada:  Well done! :bowing_man: 
+
+***
+
+## 4. Get in touch with us :mailbox_with_mail: 
+
+If you have any question, comment or suggestion, feel free to create an issue on our repo because that will be the fastest and easiest way of communicating us.
+
+If you'd prefer to email then you can contact Franklin at ffein@stanford.edu
+
+***
+
+## 5. A Special Note :high_brightness:
```

Comment:
May also be beneficial to add a section on how we [recongize contributions in BIDS](https://github.com/bids-standard/bids-specification/blob/master/CONTRIBUTING.md#recognizing-contributions)